build.gradle

```diff
@@ -224,6 +224,7 @@ project(':iceberg-api') {
     testImplementation "org.apache.avro:avro"
   }
 
+  tasks.processTestResources.dependsOn rootProject.tasks.buildInfo
```

Comment:
Is the dependency on the next line still needed?

Reply:
No it’s not. Removed it

I checked for the warning, as well as ran build / test tasks with and without clean (and existing property file) and everything was as expected.